package.json

```diff
@@ -67,7 +67,7 @@
     "prettier:ts": "prettier --parser typescript --single-quote true --trailing-comma es5 --write 'src/**/*.ts'",
     "prettier:ci": "prettier --parser typescript --single-quote true --trailing-comma es5 --list-different 'src/**/*.ts'",
     "prettier:json": "prettier --parser json --write '**/*.json'",
-    "release": "np",
+    "release": "yarn build && np",
```

Comment:
I don't think it makes much difference, but would you be cool with us putting this in the `version` script with the other build task?